### ISSUE TITLE
CP stopps not at Endpiont

### ISSUE THREAD

**Issue by benjaminpegrisch** [Courseplay/courseplay#1169]
Sorry for my bad English, i am Austrian.

My Problem ist when i set a course an let a tractor drive this, he drive with only 9kmh and at the last Waypoint (STOPP-SIGN) the tractor want to drive back to te beginning.
I have tried it with "Überführung", "kombinierter Modus" and other course varieties.
Always the same problem.

MAp: Bjornholm Standart and all other Mod MAPs i tried.
Beta Patch 2 and final 1.3. Same problem

Same problem with 4.01  an all other dev´s inkluding the last.

No more Mods in folder than CP.

Please help me, i like CP so much

**Comment by horoman** [comment]
Hi Benjamin
According to the manual of "Überführung" http://courseplay.github.io/courseplay/de/manual/index.html#driveMode5 you have to press "Stop at last point or at next trigger" in order to prevent CP to restart at the beginning.
I don't know why it is like this. For the "Überführung" I myself would prefer the standard behiviour to be that it stops at the end and I have to click if it should continue to the beginning. But this might cause problems with triggers and break points along the course...

**Comment by benjaminpegrisch** [comment]
Thats ma Problem i need cp for driving crops, so i need "kombinierter modus "

De cp dont know the Stopp sign. What can i do? 
And: does other have the.Same Problem or only i?
Log is clean. Anybody heard about.this

**Comment by benjaminpegrisch** [comment]
Problem?
When not must i delete fs15 and install new?

Need help please
Thanks for all

**Comment by horoman** [comment]
Can you please give us some more detailed information? What exactly are you trying to do? Crops. Which ones from where to where? Are you using a tractor with trailers?

**Comment by horoman** [comment]
The combi mode, as explained on http://courseplay.github.io/courseplay/de/manual/index.html#driveMode2, is used to unload a combine that is working on a field. The recorded course must(!) start and end on the field, the end point should be aligned. After the trailer is full the tractor goes to the start point and drives the recorded route to the unloading trigger, unloads, keeps following the route and returns to the end point on the field to get loaded again.
In other words going to the start point after the end point is not a bug, it's on purpose. It's needed to work properly.
Pleas make sure, you are using the mode that is intended to be used for your operation.

**Comment by benjaminpegrisch** [comment]
MUST the ending point be in the field? My start point and my ending point is on the road next to the field. Could this be a Problem?

**Comment by benjaminpegrisch** [comment]
To "überführung" Drive mode 5:

I record a course for example:

20meters then finish recording. I drive to Start point and let him drive. He drives konstant 9kmh, but i activatet max speed. So he drives with 9kmh to the end point (Stopp sign). There he doesnt Stopp because he turns left and drive back to Start point...

**Comment by benjaminpegrisch** [comment]
NEWS: I deletet the XML in the savegame, installed CP new and now drive Mode 2 does working korrektly. Only drive Mod5 "Überführung" dont stopps at stop sign!

**Comment by horoman** [comment]
>MUST the ending point be in the field? My start point and my ending point is on the road next to the field.

Yes. Otherwise the tractor will not recognize the combine. It might work if you switch combine detection to manual though.

About the transfer mode (mode 5, "Überführung"):
>20meters then finish recording.

Over the first and the last few waypoints CP drives slower. Therefore if your track is only 20m, the whole course will be driven with a slow speed. Please try a much longer course.
>to the end point (Stopp sign). There he doesnt Stopp

Please if you ask questions, read our answers before you ask again:
>you have to press "Stop at last point or at next trigger" in order to prevent CP to restart at the beginning.

Thanks for your attention.

**Comment by benjaminpegrisch** [comment]
Thanks for all your pleasure with me... This issue can be closed

**Comment by horoman** [comment]
Thank you for your final feedback. This is appreciated here.

**Comment by Lustiger-Hubi** [comment]
Ich habe das Problem
1. FS19_DondiegoMap (Version: 1.4)
2. FS19_MAN_TGS18500_570PS_730PS_1000PS (Version: 1.2)
3. Giants Krampe SB3060
Einstellung Abfahrer Beladen am Startpunkt
Durchläufe 4 Er stopp jedoch nicht nach den 4 durchlauf
![fsScreen_2019_03_24_13_02_26](https://user-images.githubusercontent.com/48583507/54879262-489e1c00-4e37-11e9-8636-38732ec81430.png)
